js-wixcode-sdk/$w/guides/Styling Elements with CSS.md

```diff
@@ -94,7 +94,141 @@ For reference, here is a list of global classes:
 - [Time picker](/time-picker/css-classes)
 - [Upload button](/upload-button/css-classes)
 - [Vector image](/vector-image/css-classes)
-- [Vertical line]($w/element/css-classes#vertical-line)
+- [Vertical line](#vertical-line)
 - [Vertical menu]($w/menu/css-classes#vertical-menu)
 - [Video box](/video-box/css-classes)
-- [Video player](/video-player/css-classes)
\ No newline at end of file
+- [Video player](/video-player/css-classes)
+
+The following classes are can be customized but don't have an associated $w API:
+
+### Image button classes
+
+#### `.image-button`
+
+Targets image button elements.
```

Comment:
When reading this, I thought that it's a compound noun "Target image button elements" with a grammar error instead of a sentence without a subject. Maybe we could use the verb affect/affects? But it could also just be me, so feel free to ignore.